databricks/koalas/series.py

```diff
@@ -468,8 +525,33 @@ def to_pandas(self):
     # Alias to maintain backward compatibility with Spark
     toPandas = to_pandas
 
-    @derived_from(pd.Series)
     def isnull(self):
+        """
+        Detect existing (non-missing) values.
+        Return a boolean same-sized object indicating if the values are NA.
+        NA values, such as None or numpy.NaN, gets mapped to True values.
+        Everything else gets mapped to False values. Characters such as empty strings '' or
+        numpy.inf are not considered NA values
+        (unless you set pandas.options.mode.use_inf_as_na = True).
+
+        Returns
+        -------
+        Series: Mask of bool values for each element in Series
+               that indicates whether an element is not an NA value.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'age': [5, 6, np.NaN],
+        ...    'born': [pd.NaT, pd.Timestamp('1939-05-27'),
+        ...             pd.Timestamp('1940-04-25')],
+        ...    'name': ['Alfred', 'Batman', ''],
+        ...    'toy': [None, 'Batmobile', 'Joker']})
+        >>> df.age.isnull() # doctest: +NORMALIZE_WHITESPACE
+        0    False
+        1    False
+        2     True
+        Name: ((age IS NULL) OR isnan(age)), dtype: bool
```

Comment:
this name is probably different from what pandas does. similar issue to https://github.com/databricks/koalas/pull/273#discussion_r282307905 ?

Reply:
Yes, I think so.